assets/js/components/DetailsPermaLinks.js

```diff
@@ -44,9 +45,10 @@ export default function DetailsPermaLinks( { title, path } ) {
 	return (
 		<Fragment>
 			<Link
-				className="googlesitekit-display-block"
-				href={ detailsURL }
+				className={ ! serviceURL ? classnames( 'googlesitekit-display-block' ) : '' }
```

Comment:
Maybe this doesn't need the class at all since the link below has it as well, they'll always be on separate lines visually. I took a quick look and it seems to look the same in both cases. Please confirm and if so, let's just remove it since it otherwise doesn't make much sense to render it conditionally.

Reply:
You are right, I have removed it 👍